vnext/Microsoft.ReactNative.IntegrationTests/TurboModuleTests.cpp

```diff
@@ -163,7 +163,7 @@ struct CppTurboModuleSpec : TurboModuleSpec {
   }
 };
 
-REACT_MODULE(CppTurboModule)
+REACT_MODULE_NOREG(CppTurboModule)
```

Comment:
How to use the unregistered module?
It would be nice to show it in the new test.

Reply:
Changing to `NOREG` only affects `TryAddAttributedModule` and `AddAttributedModule`, making the module cannot be discovered by these two functions. So the only difference is to register it manually. And in devmain and rnw predefined modules, as far as I know all are registered manually.

Reply:
Right, but we have the attributed modules to be used in other scenarios and you effectively removing these tests. While I agree that we need to test the new scenarios, I believe we must keep testing the previous scenarios instead of removing tests for them.
It does not need to be the full test. We just need keep testing the other way for module registration. I.e. It can be a simple module with just one metheod.